mu-plugins/blocks/navigation/index.php

```diff
@@ -0,0 +1,240 @@
+ * navigate off-site.
+ *
+ * @param string $menu_slug
+ *
+ * @return array|boolean List of menu items with label, url, or false if the
+ *                       slug does not match a navigation list.
+ */
+function get_menu_items( $menu_slug ) {
+	switch ( $menu_slug ) {
+		case 'about-details':
+			return array(
+				array(
+					'label' => __( 'Domains', 'wporg' ),
+					'url' => '/about/domains/',
+				),
+				array(
+					'label' => __( 'License', 'wporg' ),
+					'url' => '/about/license/',
+				),
+				array(
+					'label' => __( 'Accessibility', 'wporg' ),
+					'url' => '/about/accessibility/',
+				),
+				array(
+					'label' => __( 'Privacy Policy', 'wporg' ),
+					'url' => '/about/privacy/',
+				),
+				array(
+					'label' => __( 'Statistics', 'wporg' ),
+					'url' => '/about/stats/',
+				),
+			);
+			break;
+		case 'about-technology':
+			return array(
+				array(
+					'label' => __( 'Requirements', 'wporg' ),
+					'url' => '/about/requirements/',
+				),
+				array(
+					'label' => __( 'Features', 'wporg' ),
+					'url' => '/about/features/',
+				),
+				array(
+					'label' => __( 'Security', 'wporg' ),
+					'url' => '/about/security/',
+				),
+				array(
+					'label' => __( 'Roadmap', 'wporg' ),
+					'url' => '/about/roadmap/',
+				),
+				array(
+					'label' => __( 'History', 'wporg' ),
+					'url' => '/about/history/',
+				),
+			);
+		case 'about-people':
+			return array(
+				array(
+					'label' => __( 'Philosophy', 'wporg' ),
+					'url' => '/about/philosophy/',
+				),
+				array(
+					'label' => __( 'Etiquette', 'wporg' ),
+					'url' => '/about/etiquette/',
+				),
+				array(
+					'label' => __( 'Swag', 'wporg' ),
+					'url' => 'https://mercantile.wordpress.org/',
+				),
+				array(
+					'label' => __( 'Logos', 'wporg' ),
+					'url' => '/about/logos/',
+				),
+				array(
+					'label' => __( 'People of WordPress', 'wporg' ),
+					'url' => 'https://wordpress.org/news/category/community/',
+				),
+			);
+		case 'download':
+			return array(
+				array(
+					'label' => __( 'Releases', 'wporg' ),
+					'url' => '/download/releases/',
+				),
+				array(
+					'label' => __( 'Nightly', 'wporg' ),
+					'url' => '/download/beta-nightly/',
+				),
+				array(
+					'label' => __( 'Counter', 'wporg' ),
+					'url' => '/download/counter/',
+				),
+				array(
+					'label' => __( 'Source', 'wporg' ),
+					'url' => '/download/source/',
+				),
+			);
+		// Local navigation for wordpress.org/documentation
+		case 'documentation':
+			return array(
+				array(
+					'label' => __( 'WordPress Overview', 'wporg' ),
+					'url' => '/overview/',
+				),
+				array(
+					'label' => __( 'Technical Guides', 'wporg' ),
+					'url' => '/technical-guides/',
+				),
+				array(
+					'label' => __( 'Support Guides', 'wporg' ),
+					'url' => '/support-guides/',
+				),
+				array(
+					'label' => __( 'Customization', 'wporg' ),
+					'url' => '/customization/',
+				),
+			);
```

Comment:
New config for Documentation

Reply:
That would be ideal yes, can you point me to an example please?

Reply:
sure! Here are a few simple examples:

- [the label in Query Total](https://github.com/WordPress/wporg-mu-plugins/blob/ae99783522f0803a5ebcdd374a6ebe2ed001abe1/mu-plugins/blocks/query-total/index.php#L40-L56), then used in [this showcase PR](https://github.com/WordPress/wporg-showcase-2022/blob/822b29d8e1445bdf63329e1e7e2cd48e8cf88922/source/wp-content/themes/wporg-showcase-2022/inc/block-config.php#L16-L26)
- [the breadcrumbs in Site Breadcrumbs](https://github.com/WordPress/wporg-mu-plugins/blob/ae99783522f0803a5ebcdd374a6ebe2ed001abe1/mu-plugins/blocks/site-breadcrumbs/index.php#L63-L70)

I think this might be better suited to running over an empty array, and then bailing early if it's not updated, like [the query filters configuration](https://github.com/WordPress/wporg-mu-plugins/blob/c79d373ffdb91cd14cc12f08e23173fa872921e8/mu-plugins/blocks/query-filter/render.php#L6-L32) (again, [used in that showcase PR](https://github.com/WordPress/wporg-showcase-2022/blob/822b29d8e1445bdf63329e1e7e2cd48e8cf88922/source/wp-content/themes/wporg-showcase-2022/inc/block-config.php)).

Reply:
Maybe site-breadcrumbs is one

https://github.com/WordPress/wporg-mu-plugins/blob/trunk/mu-plugins/blocks/site-breadcrumbs/index.php#L70
https://github.com/WordPress/wporg-main-2022/blob/trunk/source/wp-content/themes/wporg-main-2022/functions.php#L24

Reply:
Snap, thank you, I knew there would be a way, and it always seems to be via filters :)

Reply:
Got this working in https://github.com/WordPress/wporg-mu-plugins/pull/443/commits/9b73837fa39c90080c8e18bc3097f7a1e4f844d4